extensions/dacpac/src/wizard/api/basePage.ts

```diff
@@ -69,11 +72,11 @@ export abstract class BasePage {
 			let srv = c.options.server;
 
 			if (!db) {
-				db = '<default>';
+				usr = localize('basePage.defaultDb', '<default>');
```

Comment:
should this variable being updated be `db`?

Reply:
oops thanks for catching this!